examples/badwords.py

```diff
@@ -0,0 +1,14 @@
+import voltage  # Import voltage.
+
+client = voltage.Client()  # Initialize the client.
+
+
+@client.listen("message")  # Listen for words.
+async def badwords(message):  # Doesn't matter what you call the function.
+    bad_words = ["fuck", "shit", "bitch", "slut"] # The words the revolt bot is looking for.
+    if any(word in message.content for word in bad_words):
+        await message.delete() # Deleting the words.
+        await message.channel.send(f"{message.author.mention} you can't say that word!") # Send a message. 
```

Comment:
why not just use `message.send`

Reply:
Because that doesn't exist I supoose.

Reply:
> Because that doesn't exist I supoose.

why not? and if it doenst then `message.delete` exists?

Reply:
> > Because that doesn't exist I supoose.
> 
> why not? and if it doenst then `message.delete` exists?

`message.delete` as you'd imagine deletes the message so on the same beat it would make sense for `message.send` to send the `Message` object the method is called on but that wouldn't be possible as `Message` objects are only constructed once the websocket receives that someone sent a message I suppose.

Making `message.send` send a new message to the messages channel wouldn't make sense and it's already doable via `message.channel.send` I suppose, alternatively you could use `message.reply` but thanks to revolt weirdness you can still see the content of a message if it was replied to in the reply if it was cached which would defeat the purpose of this particular example I suppose.

Additionally, if we look at something like discord.py they don't have a `message.send` function (atleast not that I know of) I suppose.

HOWEVER in voltage's command framework which recently got added to the main library and similarly to discord.py `ctx.send` is an alias for `ctx.channel.send` I suppose.

Making `message.send` a thing wouldn't be that hard per say, it would just require making it a subclass of `Messegable` and making the `get_id` function return the id of the message's channel but that would defeat the first point I stated and also add more stuff that's more appropriate for stuff that should contain messages (like `message.history` and such) I suppose.

I'd love to hear your opinion on this. (I suppose)